app/src/main/kotlin/com/coredocker/android/views/screen/loginscreen/LoginScreenActivity.kt

```diff
@@ -25,18 +26,16 @@ class LoginScreenActivity : AppCompatActivity() {
             this.lifecycleOwner = this@LoginScreenActivity
             this.loginViewModel = _viewModel
         }
-
-        _viewModel.isLoggedIn.observe(this, Observer { isLoggedIn ->
-            if (isLoggedIn) {
-                openHomeView()
+        _navigate.currentView.observe(this, Observer { n ->
```

Comment:
https://github.com/flatcircle/LiveDataHelper/blob/master/livedatahelper/src/main/java/io/flatcircle/livedatahelper/LiveDataExtensions.kt

Some convenience extensions to consider.

Reply:
👍 Will check it out.